### ISSUE TITLE
Unable to deploy on synology docker. Connection reset

### ISSUE THREAD

**Issue by Larosen** [fred4jupiter/fredbet#38]
With the World Cup in Qatar approaching, I would appreciate someone providing a step by step guide on how to run this container. I would prefer to run it on my synology in a container and share it with my friends from there. But maybe it can be done for free as part of the Google Cloud Free tier. 

I have absolutely no idea what needs to be done. If I load the image fred4jupiter/fredbot:latest and run that, the logs look like  this:  
<img width="1165" alt="Bildschirm­foto 2022-11-10 um 18 17 24" src="https://user-images.githubusercontent.com/66299704/201162904-6677573d-b9e3-4b9c-99f9-c8e6468afaee.png">
But at http://HOST_IP:7676/ (configuted port 7676 before) the message "The website is unreachable. Connection has been reset" comes up.

Can someone please provide me with proper instructions for "noobs"? I think this could make the code very interesting for many more people, in future tournaments.

**Comment by fred4jupiter** [comment]
Hi Larosen. The container startup looks good. I think the port mapping is wrong. The internal port is 8080 by default. What port did you configured? I didnˋt try that on a synology device, but that should be the same as on any other machine. Please have a look at the „Production Environment“ section for further information. The easiest and cheepest way to run the image in production is to run the docker compose file with Traefik and PostgreSQL in Oracle Cloud on an ARM based VM. Oracle has the Always Free Tier Option which is very nice.

**Comment by Larosen** [comment]
The Port 8080 is already in use by another container. I used Port 7676 instead. 

Your suggestion with Oracle Cloud brought the breakthrough. After almost 12 hours the website is now up and running!
However, I did not get Traefik set up in Oracle Cloud. Also the AWS BackUp had to be removed from the Docker-Compose. In the end I realized the exposion with Cloudflared via the command line. 

Thanks for this great tool, I'm excited to see what the future holds for this project!
